cmd/kaf/group.go

```diff
@@ -151,94 +154,109 @@ func createGroupCommitOffsetCmd() *cobra.Command {
 			client := getClient()
 
 			group := args[0]
+			partitionOffsets := make(map[int32]int64)
 
-			var partitions []int32
-			if allPartitions {
-				// Determine partitions
-				admin := getClusterAdmin()
-				topicDetails, err := admin.DescribeTopics([]string{topic})
-				if err != nil {
-					errorExit("Unable to determine partitions of topic: %v\n", err)
+			if offsetMap != "" {
+				match, _ := regexp.MatchString("^\\d+:\\d+(;\\d+:\\d+)*$", offsetMap)
+				if !match {
+					errorExit("Wrong --offset-map format. Use semicolon (;) divided list of single partition:offset mappings.\nExample: --offset-map 0:123;1:135;2:120\n")
```

Comment:
can we use JSOn for the offset map? for partition reassignments, we're already using JSON, i think it can make sense here as well.

Reply:
This would also make the parsing much more straight forward (json.Unmarshal)

Reply:
Sure, thanks for the feedback :+1:

Reply:
`json.Unmarhal` seems accepting only strinct JSON. It means that the parameter should be defined as in following example:

`--offset-map '{"0": 123, "1": 135, "2": 120}'`

Keys (partition numbers) have to be wrapped by quotes. Which feels strange as it has to be always numbers. Are we fine with that?

Or I can just parse it by regex again and just pretend it is JSON in special form (object where both keys and values are numbers).

Reply:
I think it is not a big deal and code is simple with JSON format. What do you think?